access_application_test.go

```diff
@@ -74,7 +75,7 @@ func TestAccessApplications(t *testing.T) {
 		CustomDenyMessage:       "denied!",
 		CustomDenyURL:           "https://www.example.com",
 		SameSiteCookieAttribute: "strict",
-		HttpOnlyCookieAttribute: true,
+		HttpOnlyCookieAttribute: &httpOnlyVal,
```

Comment:
to make life easier, i did add some type converters a little while back for this. here, we can use [`BoolPtr`](https://github.com/cloudflare/cloudflare-go/blob/master/convert_types.go#L77).

```suggestion
		HttpOnlyCookieAttribute: BoolPtr(true),
```